servers/src/main/java/tachyon/master/file/meta/Inode.java

```diff
@@ -278,6 +295,54 @@ public synchronized void setPinned(boolean pinned) {
     mPinned = pinned;
   }
 
+  /**
+   * @return the username of the inode
+   */
+  public synchronized String getUserName() {
+    return mUserName;
+  }
+
+  /**
+   * Sets the userName of the inode.
+   *
+   * @param userName the userName of the inode
+   */
+  public synchronized void setUserName(String userName) {
+    mUserName = userName;
+  }
+
+  /**
+   * @return the groupname of the inode
+   */
+  public synchronized String getGroupName() {
+    return mGroupName;
+  }
+
+  /**
+   * Sets the groupName of the inode.
+   *
+   * @param groupName the groupName of the inode
+   */
+  public synchronized void setGroupName(String groupName) {
+    mGroupName = groupName;
+  }
+
+  /**
+   * @return the permission of the inode
+   */
+  public synchronized short getPermission() {
+    return mPermission;
+  }
+
+  /**
+   * Sets the permission of the inode.
```

Comment:
the method-level comment for setter is not required as it does not add new information